### PR TITLE
Feat/1xxx place info

### PR DESCRIPTION
**Pull request by crui3er**

### CODE REVIEW

File: src/dotnet/Core/Links.cs [hunk 1]
```diff
@@ -24,6 +24,9 @@ public static LocalUrl EmbeddedChat(ChatId chatId, long? entryId = null)
             ? $"/embedded/{chatId.Value}" + ChatEntryQuery(vEntryId)
             : $"/embedded/{chatId.Value}";
 
+    public static LocalUrl PlaceInfo(PlaceId placeId)
+        => $"/place-info/{placeId.Value}";
```
Comment (frolyo) [inline src/dotnet/Core/Links.cs]: is `/place/...` taken or supposed to be taken by some other page?

File: src/dotnet/UI.Blazor.App/Components/PlaceInfo/PlaceInfoPage.razor [hunk 2]
```diff
@@ -1,61 +1,71 @@
...
-    private void OnPlaceClick(PlaceId placeId) {
-        if (placeId == PlaceId.None)
-            return;
-        _selectedPlaceId = placeId;
-        State.Recompute();
+        if (!PlaceId.TryParse(placeSid, out var placeId))
+            return Model.Unavailable;
+
+        try {
+            var place = await Places.Get(Session, placeId, cancellationToken).ConfigureAwait(false);
```
Comment (frolyo) [inline src/dotnet/UI.Blazor.App/Components/PlaceInfo/PlaceInfoPage.razor:50]: needs extract to local variable

**Review by frolyo — APPROVED** [review]

File: src/dotnet/UI.Blazor.App/Components/PlaceInfo/PlaceInfoPage.razor (hunk 2 above)
Comment (crui3er) [reply src/dotnet/UI.Blazor.App/Components/PlaceInfo/PlaceInfoPage.razor:50]: I don't get what should be extracted to local var?

File: src/dotnet/Core/Links.cs (hunk 1 above)
Comment (crui3er) [reply src/dotnet/Core/Links.cs]: Changed.

**Pull request merged**